src/main/java/uk/gov/hmcts/reform/divorce/orchestration/service/impl/CaseOrchestrationServiceImpl.java

```diff
@@ -503,6 +506,20 @@ public CcdCallbackResponse dnSubmitted(CcdCallbackRequest ccdCallbackRequest, St
         return documentGenerationWorkflow.run(ccdCallbackRequest, authToken, templateId, documentType, filename);
     }
 
+    @Override
+    public Map<String, Object> handleCostsOrderGenerationCallback(final CcdCallbackRequest ccdCallbackRequest, final String authToken,
+                                                          final String templateId, final String documentType, final String filename)
+            throws WorkflowException {
+        if (String.valueOf(ccdCallbackRequest.getCaseDetails().getCaseData()
```

Comment:
Strings literals should be placed on the left side when checking for equality

https://rules.sonarsource.com/java/tag/bad-practice/RSPEC-1132
```suggestion
        if (YES_VALUE.equalsIgnoreCase(ccdCallbackRequest
                                                       .getCaseDetails()
                                                      .getCaseData()
                                                      .get(DIVORCE_COSTS_CLAIM_CCD_FIELD))